spec/api-remote-spec.js

```diff
@@ -22,7 +27,29 @@ describe('remote module', () => {
 
   afterEach(() => closeWindow(w).then(() => { w = null }))
 
+  describe('remote.getGlobal', () => {
+    it('can return custom values', () => {
+      ipcRenderer.send('handle-next-remote-get-global')
+      expect(remote.getGlobal('test')).to.be.equal('Hello World!')
+    })
+
+    it('throws when no returnValue set', () => {
+      ipcRenderer.send('handle-next-remote-get-global')
+      expect(() => remote.getGlobal('process')).to.throw('Invalid global: process')
+    })
+  })
+
   describe('remote.require', () => {
+    it('can return custom values', () => {
+      ipcRenderer.send('handle-next-remote-require')
```

Comment:
Can we move more test logic here? I want to control how a require is handled from a test itself instead of hardcoding it in a some file out there.

```js
// test
ipcRenderer.send('handle-next-remote-require',
    {/* moduleName */ test: /* returnValue */ 'Hello World!'})
const returnedValue = remote.require('test')
expect(returnedValue).to.equal('Hello World!')

// event handler
ipcMain.on('handle-next-remote-require', function (event, modulesMap = {}) {
   event.sender.once('remote-require', (event, moduleName) => {
     if (modulesMap.hasOwnProperty(moduleName)) {
       event.preventDefault()
       event.returnValue = modulesMap[moduleName]
     }
   })
 })
```

And the same thing for the `.getGlobal()` of course.